ptx/sec_deriv_basic_rules.ptx

```diff
@@ -465,11 +478,13 @@
             <caption>A graph of <m>f(x) = \sin(x)+2x+1</m> along with its tangent line approximation at <m>x=\pi</m>, zoomed in</caption>
             <image xml:id="img_sintangapproxzoom">
               <shortdescription>
-                Highly zoomed in graph of a sine function that is increased by 2.
+                Highly zoomed in graph of a sine function that is increases by 2*x+1.
```

Comment:
Same issue here. Whether "increased" or "increases", this could be confusing to the reader.

Try this:
"A zoomed in view of the previous graph and its tangent line, near x=pi."